.github/workflows/ruby.yml

```diff
@@ -23,8 +24,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.0', '3.1']
-        activerecord: ['6.0', '6.1', '7.0', '7.1']
+        ruby-version: ['3.0', '3.1', '3.2', '3.3']
```

Comment:
Same as active-record_acts-as, please just focus the testing for
- Rails 7.0, 7.1, 7.2
- Ruby 3.1, 3.2, 3.3